frontend-react/src/shared/Card/Card.module.scss

```diff
@@ -5,24 +5,34 @@
  * with differing style expectations
  */
 .usa-card--alternate {
-    :global(.usa-card__header) {
-        @include u-padding-left(0);
-    }
-    :global(.usa-card__heading) {
-        @include u-padding-left(0);
-    }
-    :global(.usa-card__body) {
-        @include u-padding-left(0);
-    }
-    :global(.usa-card__footer) {
-        @include u-padding-left(0);
-    }
+    :global {
+        .usa-card__header {
+            @include u-padding-left(0);
+        }
 
-    :global(.usa-card__container) {
-        border-style: none;
-    }
+        .usa-card__heading {
```

Comment:
what is the difference between `.usa-card__header` and `.usa-card__heading`? can we make them semantically more meaningful?

Reply:
I didn't create them. The card component is part of uswds component library, I'm just overriding them.